ui/values/localizable/en.go

```diff
@@ -499,4 +499,8 @@ unlockWithPassword = "Unlock with password"
 "confirmVote" = "Confirm your vote"
 "policySetSuccessfully" = "Your treasury policy has been successfully updated!"
 "colon" = ": "
+"ok" = "OK"
+"dexDataReset" = "DEX client data reset complete."
+"dexDataResetFalse" = "DEX client data reset failed. Check the logs."
+"notSameAccoutMixUnmix" = ""Cannot use same account for mixed & unmixed""
```

Comment:
Please add these to the es.go and fr.go files

Reply:
it doesn't necessarily have to, we would have a translator do all non-english translations

Reply:
Alright fine
Resolve the conversations